recipes/grpc/all/conanfile.py

```diff
@@ -98,7 +98,7 @@ def validate(self):
             # libabsl_time.a(duration.cc.o): undefined reference to symbol '_ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEPKc@@GLIBCXX_3.4.21'
             if (self.settings.os == "Linux" and self.settings.compiler == "gcc") and not self.options["abseil"].shared:
                 raise ConanInvalidConfiguration(
-                    "gRPC shared not supported yet without abseil shared"
+                    "gRPC shared not supported yet without abseil static"
```

Comment:
Current message is correct, why this change?

Reply:
In the meantime, thanks to https://github.com/conan-io/conan-center-index/pull/9703, this `ConanInvalidConfiguration` may be removed.

Reply:
Ah sorry, yes, I read incorrectly, sorry wasting your time. 

Double negation 🙈...

Reply:
Maybe "with abseil static"